t/page/countries.rb

```diff
@@ -0,0 +1,26 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/countries'
+require 'pry'
+
+CJSON = 'https://cdn.rawgit.com/everypolitician/everypolitician-data/%s/countries.json'.freeze
+SHA   = 'd8a4682f'.freeze
+
+describe 'Countries' do
+  subject do
+    Page::Countries.new
+  end
+
+  it 'has a list of all countries in Everypolitician' do
+    assert_equal subject.countries.first[:name], 'Abkhazia'
```

Comment:
All the other test files use the ".must_equal" syntax rather than the `assert_equal` syntax. I don't massively care either way which we use, but we should be consistent.